src/main/scala/chisel3/util/circt/SizeOf.scala

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3.util.circt
```

Comment:
Should we call this experimental? Especially because it doesn't have firrtl compiler support yet?

Reply:
We have MFC support for this and three other intrinsics: https://github.com/llvm/circt/blob/main/lib/Dialect/FIRRTL/Transforms/LowerIntrinsics.cpp#L198

Reply:
It's been supported in circt since Dec 18 or so.